pyart/retrieve/tests/test_kdp_proc.py

```diff
@@ -7,7 +7,24 @@
 from pyart.testing import sample_objects
```

Comment:
This file should be mode 644 not 755.  The new mode results in the tests in this file being overlooked in some cases.